src/name_bazaar/ui/pages/register_name_page.cljs

```diff
@@ -42,7 +43,9 @@
           "Register"]]]])))
 
 (defmethod page :route.registrar/register []
-  [app-layout
-   [ui/Segment
-    [:h1.ui.header.padded "Register Name"]
-    [register-name-form]]])
+  [:div.wrapper
+   [name-bazaar-meta-tags {:title "Name Bazaar : Register Name" :description "Register your END domain name"}]
```

Comment:
:title "NameBazaar - Register ENS Name"
:description "Simplest way to register a new ENS name."